q2_diversity_lib/plugin_setup.py

```diff
@@ -68,7 +68,6 @@
     name='Observed Features',
     description='Compute the number of observed features for each sample in a '
                 'feature table',
```

Comment:
@ebolyen, there's a stray comma at the end of this line in the plugin registration that doesn't seem to affect the tests. Am I good to fix in my beta-div PR, or should this be addressed directly?

Reply:
Yeah I saw that, the linter passed, your call if you want to change it, but this PR is fine :)